requirements.txt

```diff
@@ -1,11 +1,14 @@
-pyqt5
-requests
-beautifulsoup4
-scandir
-rarfile
-watchdog
-robobrowser
-Send2Trash
-pillow
-python-dateutil
-QtAwesome==0.3.3
\ No newline at end of file
+--index-url https://pypi.python.org/simple/
```

Comment:
>As far as I know, unless @Pewpews doesn't plan on using PyPI for distribution.

sorry i don't understand this and connection with this line.

>--index-url https://pypi.python.org/simple/

i know pip have index-url flag, but i don't know if there is need to change the default Python Package Index